### PR TITLE
Improve readability of Scala3 detection

### PR DESCRIPTION
**Pull request by mdedetrich**
Very minor change but this makes code easier to read.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -5,7 +5,7 @@ val Scala2_13 = "2.13.10"
 val Scala3    = "3.2.2"
 
 val isScala3 = Def.setting(
-  CrossVersion.partialVersion(scalaVersion.value).exists(_._1 == 3)
+  CrossVersion.partialVersion(scalaVersion.value).exists { case (epic, _) => epic == 3 }
```
Comment (sirthias) [inline build.sbt:8]: "epic" is nice! :)
I think you mean "epoch", right?
(Let's leave _epic_ though, I like it... :)

Comment (mdedetrich) [reply build.sbt:8]: I thought the technical term for Scala3 style versioning is epic.major.minor but that might just be my brain playing up. I can change it to epoch if you want.

Comment (sirthias) [reply build.sbt:8]: No, "epic" is better... Thank you!

Comment (xuwei-k) [reply build.sbt:8]: we can use `scalaBinaryVersion.value == “3”`

Comment (mdedetrich) [reply build.sbt]: That is much nicer, done

**Pull request merged**